src/bitcoin-cli.cpp

```diff
@@ -380,14 +380,12 @@ static UniValue CallRPC(BaseRequestHandler *rh, const std::string& strMethod, co
 
     // check if we should use a special wallet endpoint
     std::string endpoint = "/";
-    if (!gArgs.GetArgs("-rpcwallet").empty()) {
-        std::string walletName = gArgs.GetArg("-rpcwallet", "");
-        char *encodedURI = evhttp_uriencode(walletName.data(), walletName.size(), false);
+    if (rpcwallet) {
```

Comment:
I think you should also include release notes to this PR (this backport):
    Merge #19354: doc: add release note for -getinfo displaying multiwallet balances

Reply:
Resolved in latest push, good catch!